pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -629,13 +651,72 @@ func (b *OFBridge) RetryInterval() time.Duration {
 	return b.retryInterval
 }
 
+func (b *OFBridge) queryTableFeatures() {
+	mpartRequest := &openflow13.MultipartRequest{
+		Header: openflow13.NewOfp13Header(),
+		Type:   openflow13.MultipartType_TableFeatures,
+		Flags:  0,
+	}
+	mpartRequest.Header.Type = openflow13.Type_MultiPartRequest
+	mpartRequest.Header.Length = mpartRequest.Len()
+	tableFeatureChan := make(chan *openflow13.MultipartReply)
```

Comment:
bit: should this be a buffered channel? asking because we have a `b.ofSwitch.Send` in the loop, and I don't remember if that can take time or not. If it can, probably better to have a buffered channel which can keep storing multipart fragments.

Reply:
Greate catch, `Send` is possible to take some time if many messages are expected to be sent (might not only Multipart messages, also including other messages like FlowMod). I will add a buffer.